pyproject.toml

```diff
@@ -12,6 +12,7 @@ pandas = "^2.2.1"
 unidecode = "^1.3.8"
 xmltodict = "^0.13.0"
 dpytools = {git = "https://github.com/ONSdigital/dp-python-tools.git"}
+dictdiffer = "0.9.0"
 
 [tool.poetry.group.dev.dependencies]
```

Comment:
Move dictdiffer into `[tool.poetry.group.dev.dependencies]` instead of `[tool.poetry.dependencies]`